storage/spider/mysql-test/spider/bugfix/t/mdev_24020.test

```diff
@@ -0,0 +1,97 @@
+--source ../include/mdev_24020_init.inc
+--echo
+--echo this test is for MDEV-24020
+--echo
+--echo drop and create databases
+
+--connection master_1
+--disable_warnings
+CREATE DATABASE auto_test_local;
+USE auto_test_local;
+
+--connection child2_1
+SET @old_log_output = @@global.log_output;
+SET GLOBAL log_output = 'TABLE,FILE';
```

Comment:
I may miss something, but why do we need to set `log_output`?

Reply:
You're right. There is no need to set `log_output`.
I copied the spider test template from `mdev_20502.test`, but I forgot to remove these lines. Sorry for making you confused.

Reply:
OK. Other queries concerning the general log might also be redundant.